server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -14,23 +14,25 @@ use crate::sync::{
 use super::{
     sync_api_v3::SyncApiV3,
     sync_api_v5::{RemoteSyncActionV5, RemoteSyncBatchV5},
-    SyncApiV5,
+    SyncApiV5, SyncConnectionError,
 };
 
 #[derive(Error, Debug)]
-#[error("{msg}: {source}")]
-pub struct RemoteSyncError {
-    msg: &'static str,
-    source: anyhow::Error,
-}
-
-impl From<RepositoryError> for RemoteSyncError {
-    fn from(err: RepositoryError) -> Self {
-        RemoteSyncError {
-            msg: "Internal DB error",
-            source: anyhow::Error::from(err),
-        }
-    }
+#[error("Failed to send initialisation request: {0:?}")]
+pub(crate) struct PostInitialisationError(SyncConnectionError);
+#[derive(Error, Debug)]
+#[error("Failed to set initialised state: {0:?}")]
+pub(crate) struct SetInitialisedError(RepositoryError);
+#[derive(Error, Debug)]
+pub(crate) enum RemotPullError {
```

Comment:
Remot -> Remote

Reply:
Thanks, 9f6ad1091b49378a629c36298fe2d171fd020ee3 in https://github.com/openmsupply/open-msupply/issues/338